test/e2eTests/tests/login_test.js

```diff
@@ -0,0 +1,47 @@
+
+Feature('Login');
+
+Scenario('Login successful with admin credentials and logout', (I, LoginPage, Data) => {
+	// Log the user in
+	const loginStatus = LoginPage.login(Data.users.admin)
+	
+	// Validate that login was successful
+	const dashboardPage = loginStatus.successful()
+	
+	// Log the user out
+	const logoutStatus = dashboardPage.logout()
+	
+	// Validate that logout was successful
+	logoutStatus.successful()
+})
+
+Scenario('Attempt to login with a user that does not have the provider role', async (I, Apis, LoginPage, Data) => {
+	// Create a user that does not have the provider role
+	await Apis.user.create(Data.users.nonProvider)
```

Comment:
Should we remove the user after the test runs?

Reply:
@ynurmahomed @edrisse take a look at hooks.js. After every test all data that was injected is automatically removed so tests don't have to worry about doing it themselves.